graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -80,6 +81,12 @@ class CodeGen(private val config: CodeGenConfig) {
     }
 
     private fun generateForSchema(schema: String): CodeGenResult {
+        val SDL_MAX_ALLOWED_SCHEMA_TOKENS: Int = Int.MAX_VALUE
+        val parserOptions = ParserOptions.getDefaultParserOptions()
+            .transform {
+                it.maxTokens(SDL_MAX_ALLOWED_SCHEMA_TOKENS)
+            }
+        ParserOptions.setDefaultParserOptions(parserOptions)
```

Comment:
This would work, but it relies on setting the global defaultParserOptions.

Instead, I think you can just pass your `parserOptions` instance directly if you change the next line to:

```java
document = new Parser().parseDocument(schema, parserOptions);
```

Reply:
You're right, thanks for the suggestion. Will update that.

Reply:
Actually, for some reason, setting it directly using parseDocument doesn't seem to take effect 🤷‍♀️

Reply:
Weird.  Whatever works. :)